### ISSUE TITLE
Duplicate messages from multiple jobs when using tox.ini config

### ISSUE THREAD

**Issue by eugene-kulak** [PyCQA/pylint#3762]
### Steps to reproduce
```ini
##############################################################################
# PYLINT SETTINGS                                                            #
##############################################################################
[MASTER]
# Use multiple processes to speed up Pylint.
jobs=2
```
```bash
ls package3
__init__.py module.py   module2.py  module3.py
```
module.py
```python
import os


class Test:
    def do_1(self):
        pass

    def do_2(self):
        pass

    def do_3(self):
        pass

    def do_4(self):
        pass

    def do_5(self):
        pass

    def do_6(self):
        pass

    def do_7(self):
        pass

    def do_8(self):
        pass

    def do_9(self):
        pass
```
module2.py
```python
import os


class Test:
    def do(other):
        print('here')

```
module3.py
```python
import os

```
### Current behavior
produce duplicate of error for each job
```bash
pylint --rcfile=tox.ini package3
************* Module package3.module2
package3/module2.py:5:4: E0213: Method should have "self" as first argument (no-self-argument)
package3/module2.py:5:4: E0213: Method should have "self" as first argument (no-self-argument)

------------------------------------------------------------------
Your code has been rated at 8.00/10 (previous run: 8.86/10, -0.86)
```
### Expected behavior
no duplicates
```bash
pylint --rcfile=tox.ini package3
************* Module package3.module2
package3/module2.py:5:4: E0213: Method should have "self" as first argument (no-self-argument)

------------------------------------------------------------------
Your code has been rated at 8.00/10 (previous run: 8.86/10, -0.86)
```
### pylint --version output
```
pylint 2.5.3
astroid 2.4.2
Python 3.6.2 (v3.6.2:5fd33b5926, Jul 16 2017, 20:11:06)
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)]
```

this issue is really annoying

**Comment by scop** [comment]
Likely duplicate of #3584 and not limited to tox.

**Comment by AWhetter** [comment]
Closing as a duplicate of #3584. Good spot @scop
